src/components/Login.jsx

```diff
@@ -1,30 +1,30 @@
 import React from 'react';
 import NavBar from '../components/Navbar'
-import EventService from "./EventService";
+import EventService from "../services/EventService";
 
-class LoginPage extends React.Component {
+class Login extends React.Component {
     constructor(props) {
         super(props);
-        this.state = {email: '', password: '', isAuthenticated: false};
+        this.state = {email: '', password: '', isAuthenticated: false}; // NOTE: user should always be intended to be
+                                                                        // de-authenticated when they redirect to login page
         this.handleSubmit = this.handleSubmit.bind(this);
-        this.onEmailChange = this.onEmailChange.bind(this);
-        this.onPasswordChange = this.onPasswordChange.bind(this);
     }
 
-    onPasswordChange(e) {
-        this.setState({password: e.target.value});
-    }
 
-    onEmailChange(e) {
-        this.setState({email: e.target.value});
-    }
 
     handleSubmit(event){
         event.preventDefault();
-        EventService.auth.userLogin(this.state, (authSuccess) => {
+
+        const email = Document.getElementById("email").value
+        const password = Document.getElementById("password").value
```

Comment:
Are you sure this works in React? Also isn't it lowercase `document`

Reply:
Yea that's true actually, it doesn't work. I will change it back to the onEventChange listeners